apps/qubit/modules/informationobject/actions/multiFileUploadAction.class.php

```diff
@@ -44,15 +44,15 @@ public function execute($request)
     }
 
     // Add javascript libraries
-    $this->response->addJavaScript('/vendor/yui/logger/logger', 'last');
-    $this->response->addJavaScript('/vendor/yui/uploader/uploader-min', 'last');
+    $this->response->addJavaScript('/vendor/uppy/uppy-bundle.js', 'last');
+    $this->response->addStylesheet('/vendor/uppy/uppy-bundle.css', 'last');
     $this->response->addJavaScript('multiFileUpload', 'last');
```

Comment:
Could we remove these from here and add them to the view.yml file for consistency? I could see a point if they were optional, like the fullwidth treeview.

Reply:
Good point. These were historically in this file... I have moved these to the `view.yml` file.